src/apiserver/api_server_test.cc

```diff
@@ -178,21 +178,19 @@ TEST_F(APIServerTest, query) {
         {
             "code": 0,
             "msg": "ok",
-            "result": {
-                "schema": ["Int32", "String"],
-                "data": [[1, "bb"]]
-            }
+            "schema": ["Int32", "String"],
```

Comment:
```json
data: {
  schema
   data
}
```
should be more reasonable, a sql stmt not always has return rows

Reply:
query that does not return rows will not have `schema` and `data` field

Reply:
you might to check both `schema` and `data` to ensure whether rows returned. 

Another reason to use later one is to be consistent with feature extract api: https://docs.google.com/spreadsheets/d/1HWmU4WDjTWmB_OC-PyKebulAuYtTsuLJPn76dADwqBw/edit?usp=sharing